README.adoc

```diff
@@ -1,9 +1,9 @@
-= Audio Library for Arduino =
-
-The Audio library enables an Arduino Zero or MKR1000 board to play back .wav files from a storage device like an SD card.
+= AudioZero Library for Arduinos based on SAMD Architecture =
 
+The AudioZero library enables an Arduino Zero or MKR1000 board (SAMD Architecture) to play back .wav files from a storage device like an SD card.
+The Zero and the MKR1000 use the DAC0 pin to play sounds.
```

Comment:
```suggestion
The AudioZero library enables an Arduino SAMD Board (e.g., [MKR boards](https://store.arduino.cc/catalogsearch/result/?q=mkr), [Nano 33 IoT](https://store.arduino.cc/arduino-nano-33-iot), [Zero](https://store.arduino.cc/arduino-zero)) to play back .wav files from a storage device like an SD card to the `DAC0`/`A0` pin.
```
This library may be used with any SAMD board. Since there are many of these boards now, and perhaps more over time, a list of the specific boards it applies to is not a good idea.